chore/travis/travis-spoon-bytecode-verify.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# This script intends to be run on TravisCI
+# it runs verify and site maven goals
+# and to check documentation links
+
+# fails if anything fails
+set -e
+
+source /opt/jdk_switcher/jdk_switcher.sh
+
+pip install --user CommonMark==0.7.5 requests pygithub
```

Comment:
Why do you need that?

Reply:
That's a copy from `chore/travis/travis-verify.sh`, shall I remove it?

Reply:
I think we needed it for check links but apparently it's removed from the script. So I think it can be removed yes.